### ISSUE TITLE
Installing Saktumiva (was: I've got Apache2 and Dokuwiki running ... 

### ISSUE THREAD

**Issue by wujastyk** [chchch/upama#67]
I've got Apache2 and Dokuwiki running and I've copied the Saktumiva files to the /var/www/dokuwiki/lib/tpl and /var/www/dokuwiki/lib/plugins directories as instructed, and a blank sidebar.txt file in /var/www/dokuwiki/data/pages.

The result is here: http://206.12.96.209/doku.php?id=wiki:welcome

What should I do next?

Charles, I know you're not a general support desk for computer help  :-)  So you can just say no, or point me to documentation.  Or perhaps I should be asking Peter, since he's done all this successfully.

_Originally posted by @wujastyk in https://github.com/chchch/upama/issues/64#issuecomment-1785855884_

**Comment by wujastyk** [comment]
I did absolutely nothing, but suddenly I have this!
![image](https://github.com/chchch/upama/assets/762246/fe27bf4d-8242-4a38-a6ee-3c60dc3d4070)

I feel that I'm on my way at last.

**Comment by ppasedach** [comment]
The connection times out from here. Is the machine you're installing it reachable from outside your network? Other than that, from your screenshot it looks like you're on your way indeed. So what is still missing or not working?

**Comment by wujastyk** [comment]
Thanks, I think I've opened HTTP to the world again, so I'd be grateful if you would confirm that http://206.12.96.209/ works.

Assuming it does, I think I mainly need to work out a bit more about Dokuwiki settings.  Where do I put files?  Where do I make subdirectories for different editing project, and so on.  Basic stuff.  I can surely work this out, but I'm only able to give this little bursts of time here and there.

**Comment by chchch** [comment]
It seems to work! You can probably just copy your whole directory structure over to the `/data/pages/wiki/`.

**Comment by wujastyk** [comment]
Yes, it's beginning to be something I might actually be able to work with.  In any case, I'm really excited to have been able to do this.  Thanks for that directory pointer.  I also had to chown and chmod the directories under ../data to my user ID so it's writable for ftp transfers.  
The VM has 8 cpus and 16 GB RAM.  So it's like an ordinary modern PC, more or less.  I am unfortunately getting "problem loading page" when I try to collate the larger files.  
Also, putting start.txt in the data directory [displays it as plain text.](http://206.12.96.209/doku.php?id=wiki:susrutaproject:start)  We had that problem before with your Saktumiva.org; what was the answer?

**Comment by wujastyk** [comment]
With my Saktumiva installation, the blue sidebar shows no files for collation, at first.  If I load each MS transcription file manually (go wiki/<project>/<filename> for each file), then they show up in the blue sidebar and can be collated.

How can I make the files show up automatically?

**Comment by chchch** [comment]
Hmm... I haven't actually figured that out... but at least you only have to do it once...

**Comment by wujastyk** [comment]
I've now done another experiment and installed Saktumiva on my laptop.  It's been a struggle, but it's working fine now.  There are so many little issues about file permissions, apache2 settings, and so on.  One thing is still annoying me.  I have my project files in the user area of my disk; I want to create symlinks to the `/var/www/html/dokuwiki/data/pages/wiki/` directory, but when I do that, Dokuwiki sees the filenames but can't open the files.  I've tried lots of ideas, permissions, ownerships, tweaking apache2's configuration, but no luck yet.  There's documentation out there that says Dokuwiki *can* work with symlinked files, but I'm damned if I can make it work.  Hard symlinks work, but for files only, not directories.

Any ideas welcome.

My notes on setup are [here](https://cikitsa.blogspot.com/2024/05/making-your-own-saktumiva-server-using.html), if anyone wants them.
